.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+Part of 
```

Comment:
I see we don't have the "Part of " in sonarcloud-webapp PR: https://github.com/SonarSource/sonarcloud-webapp/pull/1174/files

Is the parsing gonna work fine with it here?

Reply:
It will work just fine, the "Part of " is just cosmetic help, it doesn't have any functionality. You can mention the parent issue ID in any form anywhere in the description...

I don't see a good reason why it's missing there, I'll add it

Reply:
Added & pushed